rasa/engine/storage/storage.py

```diff
@@ -54,7 +54,7 @@ def from_model_archive(
         """
         ...
 
-    @classmethod
+    @classmethod  # type: ignore[empty-body]
```

Comment:
Maybe proper fix will be to convert this method to abstract, but I don't want to break the API with dependency porting

Reply:
@m-vdb Appealing to your wisdom here 🧙🏻

Reply:
yes it looks like this method needs to be decorated with `@abc.abstractmethod`. It seems that mypy implemented more checks in the new version (see [this PR](https://github.com/python/mypy/issues/2350)). If it breaks, we'll know about it in the CI I think

Reply:
ack, will implement and check the CI